native/dll/Win32DLL/WMATagParser.cpp

```diff
@@ -148,17 +148,17 @@ class CWMProp : public IWMReaderCallback
 			switch(attType)
 			{
 			case WMT_TYPE_QWORD:
-				_ui64tow( *((QWORD*)pbValue), attValueName, 10);
+				_ui64tow_s( *((QWORD*)pbValue), attValueName, sizeof(attValueName), 10);

```

Comment:
You can't use sizeof(attValueName). Just set a variable to be cbValue + 64 and reuse that for the allocation and in these calls.

Reply:
I'll get these sorted in my head eventually :)
Done